libnetdata/libnetdata.c

```diff
@@ -1491,3 +1491,41 @@ char *read_by_filename(char *filename, long *file_size)
         *file_size = size;
     return contents;
 }
+
+//https://www.geeksforgeeks.org/c-program-replace-word-text-another-given-word/
+char *find_and_replace(const char* s, const char* find, const char* replace)
```

Comment:
what is the license here?

Reply:
hmm we will have to figure this out before merge can take place

Reply:
OK, their policy is a bit fuzzy.... Will have a look, maybe for something clearer (in licence) or from scratch.. Thanks Timo!